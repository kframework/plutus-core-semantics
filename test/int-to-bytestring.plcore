(program

(module Foo (imported) (exported () ()) (local () ())
  (define bs1 (builtin integerToByteString 1128))
  (define bs2 (builtin integerToByteString 9))
  (define bs3 (builtin integerToByteString 15))
  (define bs4 (builtin integerToByteString 16))
  (define bs5 (builtin integerToByteString 17))
  (define bs6 (builtin integerToByteString 37858))
  (define bs7 (builtin integerToByteString 0)))

)
