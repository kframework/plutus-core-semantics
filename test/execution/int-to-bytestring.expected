<T>
  <k>
    .
  </k>
  <lenv>
    .Map
  </lenv>
  <genv>
    Foo . bs1 |-> #468
    Foo . bs2 |-> #9
    Foo . bs3 |-> #F
    Foo . bs4 |-> #10
    Foo . bs5 |-> #11
    Foo . bs6 |-> #93E2
    Foo . bs7 |-> #0
  </genv>
  <modules>
    ModuleCellSetItem ( <Module>
      <curr>
        0
      </curr>
      <name>
        Foo
      </name>
      <dataCons>
        .TyExpList
      </dataCons>
      <decl>
        .QualCList
      </decl>
      <importedModules>
        .ModNmList
      </importedModules>
      <exportTy>
        .TyExpList
      </exportTy>
      <exportTm>
        .QualCList
      </exportTm>
    </Module> )
  </modules>
</T>
